charts/gateway-helm/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: gateway-helm
+name: envoy-gateway
```

Comment:
we cannot change this as we store chart in OIC registry, this's the image name.

Reply:
Hmm, OK, understand. But it's misleading.